nano/node/lmdb.cpp

```diff
@@ -1243,9 +1243,9 @@ void nano::mdb_store::upgrade_v12_to_v13 (size_t const batch_size)
 				}
 				auto block (block_get (transaction, hash, &sideband));
 				assert (block != nullptr);
-				if (sideband.height == std::numeric_limits<uint64_t>::max ())
+				if (sideband.height == 0)
 				{
-					sideband.height = height;
+					sideband.height = height + 1;
```

Comment:
Probably just make start height = 1?
`uint64_t height (1);`

Reply:
Sure, if you think that is clearer